src/main/resources/io/jenkins/plugins/checks/github/status/GitHubSCMSourceStatusChecksTrait/help-skipNotifications.html

```diff
@@ -0,0 +1,5 @@
+<div>
+    If this option is checked, the notifications sent by the <a href="https://plugins.jenkins.io/github-branch-source/">GitHub Branch Source Plugin</a>
+    will be disabled. Note that, whether checked or not, this option will only be applied when no other plugins are disabling
+    the notifications.
```

Comment:
I don't think the disclaimer is needed, it's what I would expect happens
```suggestion
    will be disabled.
```